content/en/docs/tasks/observability/distributed-tracing/opencensusagent/index.md

```diff
@@ -0,0 +1,262 @@
+---
+title: OpenCensus Agent
+description: Learn how to configure the proxies to send OpenCensus-formatted spans to OpenTelemetry Collector.
+weight: 10
+keywords: [telemetry,tracing,opencensus,opentelemetry,span,port-forwarding]
+aliases:
+    - /docs/tasks/opencensusagent-tracing.html
+owner: istio/wg-policies-and-telemetry-maintainers
+test: yes
```

Comment:
You have `yes` here, and it does generate the snips.sh, but there are no tests in this PR.

Reply:
will add tests and rewrite installation to use already existing istio samples